modules/core/shared/src/main/scala/gem/enum/GnirsCamera.scala

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem
+package enum
+
+import cats.syntax.eq._
+import cats.instances.string._
+import gem.util.Enumerated
+
+/**
+ * Enumerated type for GNRIS Camera.
+ * @group Enumerations (Generated)
```

Comment:
This is unrelated to this PR but in ocs3 i have seen errors in scaladoc with these custom annotations. or are we using a custom set?

Reply:
This is built into scaladoc but you have to pass a flag to turn it on. See https://github.com/gemini-hlsw/gem/blob/master/build.sbt#L162

Reply:
Is there a list of those. In ocs3 we fail with `@typeclass`

Reply:
http://docs.scala-lang.org/overviews/scaladoc/for-library-authors.html